NearbyMessagesDevices/.google/packaging.yaml

```diff
@@ -18,24 +18,13 @@
 # This file is used by Google as part of our samples packaging process.
 # End users may safely ignore this file. It has no relevance to other systems.
 ---
-# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
 status:       PUBLISHED
-
-# Optional, put additional explanation here for DEPRECATED or SUPERCEDED.
-# statusNote:
-
-# See http://go/sample-categories
 technologies: [Android]
 categories:   [Connectivity]
 languages:    [Java]
 solutions:    [Mobile]
-
-# May be omitted if unpublished
 github:       android/connectivity
-
-# Values: BEGINNER | INTERMEDIATE | ADVANCED | EXPERT
 level:        ADVANCED
-
 apiRefs:
     - google:com.google.android.gms.nearby.Nearby
```

Comment:
All gms core API should be listed like this:

    - gms:com.google.android.gms.location.FusedLocationProviderClient